MarsQA-1/SpecflowPages/Pages/LanguagesPage.cs

```diff
@@ -0,0 +1,304 @@
+﻿using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+
+namespace MarsQA_1.Pages
+{
+    public static class LanguagesPage
+    {
+        private static IWebElement AddNewBtn => Driver.driver.FindElement(By.XPath("//DIV[@class='ui teal button '][text()='Add New']"));
+        private static IWebElement AddLanguageTxt => Driver.driver.FindElement(By.Name("name"));
+        private static IWebElement LanguageLevelDdn => Driver.driver.FindElement(By.Name("level"));
+        private static IWebElement AddBtn => Driver.driver.FindElement(By.XPath("//INPUT[@value='Add']"));
+        private static IWebElement PopUpMsg => Driver.driver.FindElement(By.XPath("//DIV[@class='ns-box-inner']"));
+        private static IWebElement LanguageLbl => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td"));
+        private static IWebElement EditIcn => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td[3]/span[1]/i"));
+        private static IWebElement UpdateBtn => Driver.driver.FindElement(By.XPath("//input[@class='ui teal button'][@value='Update']"));
+        private static IWebElement DeleteIcn => Driver.driver.FindElement(By.XPath("//i[@class='remove icon']"));
+        private static IWebElement LanguageLevelLbl => Driver.driver.FindElement(By.XPath("//table[@class='ui fixed table']/tbody/tr/td[2]"));
+        private static ReadOnlyCollection<IWebElement> LanguageRecords => Driver.driver.FindElements(By.XPath("//div[@data-tab='first']/div/div[2]/div/table[@class='ui fixed table']/tbody"));
+        private static IWebElement CancelBtn => Driver.driver.FindElement(By.XPath("//INPUT[@class='ui button'][@value='Cancel']"));
+        private static ReadOnlyCollection<IWebElement> AddNewFields => Driver.driver.FindElements(By.XPath("//DIV[@class='fields']"));
+        private static IWebElement LanguageTab => Driver.driver.FindElement(By.XPath("//a[@data-tab='first']"));
```

Comment:
you can choose to do Driver.driver to access the driver or if that class was not static then you would need to extend classes to Driver class, can you please extend this where you need the driver

Reply:
Extended the Driver class.